pom.xml

```diff
@@ -2721,6 +2722,19 @@
       </activation>
     </profile>
 
+    <profile>
+      <id>flink1.19</id>
+      <properties>
+        <orc.flink.version>1.5.6</orc.flink.version>
+        <flink.avro.version>1.11.1</flink.avro.version>
+        <flink.format.parquet.version>1.13.1</flink.format.parquet.version>
+      </properties>
+      <activation>
+        <property>
+          <name>flink1.19</name>
+        </property>
+      </activation>
+    </profile>
```

Comment:
The 1.18 profile also needs to be fixed.

Reply:
We need to supplement all these properties for profile 1.18:

```java
        <flink.version>${flink1.17.version}</flink.version>
        <hudi.flink.module>hudi-flink1.17.x</hudi.flink.module>
        <flink.bundle.version>1.17</flink.bundle.version>
        <orc.flink.version>1.5.6</orc.flink.version>
        <flink.avro.version>1.11.1</flink.avro.version>
        <flink.format.parquet.version>1.12.3</flink.format.parquet.version>
        <flink.connector.kafka.version>${flink1.17.version}</flink.connector.kafka.version>
```

Reply:
My mistake. I keep the flink.connector.kafka.version as ${flink1.17.version}.

Reply:
Updated the Kafka connector version.